src/components/OrderBookChart.stories.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { useMemo } from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+import OrderBookChart, { OrderBookChartProps } from './OrderBookChart'
+import { Network, TokenDetails } from 'types'
+import realData from 'storybook/USDC-DAI_OrderBook_sample.json'
+import sampleDataSet from 'storybook/orderbookSamples'
+import { processRawApiData } from './OrderBookWidget'
+import { RawApiData } from 'api/dexPriceEstimator/DexPriceEstimatorApi'
+
+type SampleData = typeof sampleDataSet[0]
+
+const networkIds = Object.values(Network).filter(Number.isInteger)
+
+const defaultNetworkId = Network.Mainnet
+// All Default Tokens
+const tokenList: TokenDetails[] = CONFIG.initialTokenList.map(
+  ({ id, name, symbol, addressByNetwork, decimals = 18 }) => ({
+    id,
+    name,
+    symbol,
+    address: addressByNetwork[defaultNetworkId] || '0x',
+    decimals: decimals,
+  }),
+)
+
+// Default params to be used as initial values
+// and when there's no Token found for symbol
+const defaultParams: Omit<OrderBookChartProps, 'data'> = {
+  baseToken: {
+    id: 1,
+    name: 'Base Token',
+    symbol: 'BASE',
+    address: '0x1',
+    decimals: 18,
+  },
+  quoteToken: {
+    id: 2,
+    name: 'Quote Token',
+    symbol: 'QUOTE',
+    address: '0x2',
+    decimals: 18,
+  },
+  networkId: defaultNetworkId,
+}
+
+// Token symbols to use in control selector
+const configTokenSymbols = [defaultParams.baseToken, defaultParams.quoteToken, ...tokenList].map(
+  (token) => token.symbol || token.address,
+)
+
+export default {
+  title: 'Components/OrderBookChart',
+  component: OrderBookChart,
+  argTypes: {
+    networkId: { control: { type: 'inline-radio', options: networkIds } },
+    baseToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    quoteToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    name: { control: null },
+    description: { control: null },
+    data: { control: null },
+  },
+  decorators: [(Story): JSX.Element => <div style={{ height: '97vh' }}>{Story()}</div>],
+} as Meta
+
+//-------------------REAL-DATA------------------
+
+// Template for real data
+// fetched from price-estimation service
+const RealTemplate: Story<OrderBookChartProps> = (args) => <OrderBookChart {...args} />
```

Comment:
Why "RealTemplate", either Template or OrderBookTemplate

Also, I'm confused why `realData` is call liked that. It's just some mock data, right?

Reply:
No, real data from https://dex-price-estimator.gnosis.io/api/v1/markets/7-4?atoms=true&hops=2